src/serialize.js

```diff
@@ -94,7 +94,7 @@ jQuery.fn.extend({
 			var type = this.type;
 
 			// Use .is( ":disabled" ) so that fieldset[disabled] works
-			return this.name && !jQuery( this ).is( ":disabled" ) &&
+			return this.name && !jQuery.find.matchesSelector( this, ":disabled" ) &&
```

Comment:
I don't care about performance, just the size difference on gzipped jquery.min.js (visible as the last output from running `grunt`).